### PR TITLE
[README] Include our project goals

### PR DESCRIPTION
**Pull request by kylef**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,36 @@ resolves dependencies between libraries, fetches source code for the
...
-
 Installing and updating CocoaPods is very easy. Don't miss the [Installation
 guide](http://guides.cocoapods.org/using/getting-started.html#installation) and the
 [Getting started guide](https://guides.cocoapods.org/using/getting-started.html).
 
+## Project Goals
+
+Ultimately, the goal of CocoaPods is to improve discoverability of, and
+engagement in, third party open-source libraries. The following list
+includes the project goals which has, and continues to influence the
```
Comment (segiddins) [inline README.md]: `s/has/have/`

File: README.md [hunk 2]
```diff
@@ -13,13 +13,36 @@ resolves dependencies between libraries, fetches source code for the
...
+
+- Being able to work in the system without creating extra work for
+  library authors. Making it possible to maintain a simple transparent
+  podspec outside of the library’s repository.
+- Not imposing any judgement of ‘correctness’ on how authors decide to
+  structure their library.
+- CocoaPods should not impose any specific requirements on source-code
+  management systems. (Currently supported are git, svn, mercurial, bazaar, and
+  various types of archives downloaded over HTTP.)
+- Providing the ability to work in a distributed way, but also provide
```
Comment (segiddins) [inline README.md]: provide vs providing?

File: README.md [hunk 3]
```diff
@@ -13,13 +13,36 @@ resolves dependencies between libraries, fetches source code for the
...
+  features only possible with a centralised solution to foster a community.
+- Being able to build tools on top of the system, including those typically
+  deployed to other operating systems, such as web-services.
+- Providing opinionated and automated integration, but making that completely
+  optional. It’s perfectly possible to manually integrate the CocoaPods
+  Xcode project as you see fit, with or without workspace.
```
Comment (segiddins) [inline README.md]: workspace_s_.

File: README.md [hunk 4]
```diff
@@ -13,13 +13,36 @@ resolves dependencies between libraries, fetches source code for the
...
+- Not to depend on Apple to fix issues with Xcode or otherwise force
```
Comment (segiddins) [inline README.md]: `Not depending`

File: README.md [hunk 5]
```diff
@@ -13,13 +13,36 @@ resolves dependencies between libraries, fetches source code for the
...
+  author’s to have to do a lot of Xcode work not related to their
```
Comment (segiddins) [inline README.md]: `authors`

File: README.md (hunk 2 above)
Comment (kylef) [inline README.md]: Which one, the one at the start to near the end of the sentence?

File: README.md [hunk 6]
```diff
@@ -13,13 +13,36 @@ resolves dependencies between libraries, fetches source code for the
...
-Ultimately, the goal is to improve discoverability of, and engagement in, third
-party open-source libraries, by creating a more centralized ecosystem.
-
 Installing and updating CocoaPods is very easy. Don't miss the [Installation
 guide](http://guides.cocoapods.org/using/getting-started.html#installation) and the
 [Getting started guide](https://guides.cocoapods.org/using/getting-started.html).
 
+## Project Goals
+
+Ultimately, the goal of CocoaPods is to improve discoverability of, and
```
Comment (segiddins) [inline README.md]: `improve the discoverability of`

**Comment by segiddins** [comment]
:+1:

**Comment by orta** [comment]
shippit

**Comment by alloy** [comment]
What about flipping 'discoverability' and 'engagement'? Now it may seem like discoverability is more important.

File: README.md [hunk 7]
```diff
@@ -13,13 +13,36 @@ resolves dependencies between libraries, fetches source code for the
...
-
 Installing and updating CocoaPods is very easy. Don't miss the [Installation
 guide](http://guides.cocoapods.org/using/getting-started.html#installation) and the
 [Getting started guide](https://guides.cocoapods.org/using/getting-started.html).
 
+## Project Goals
+
+Ultimately, the goal of CocoaPods is to improve the discoverability of,
+and engagement in, third party open-source libraries. The following list
+includes the project goals which have influenced, and continues to
```
Comment (alloy) [inline README.md]: Singular 'continue' instead?

**Comment by alloy** [comment]
Thanks!

**Pull request merged**